detections/endpoint/multiple_users_failing_to_authenticate_from_process.yml

```diff
@@ -0,0 +1,68 @@
+name: Multiple Users Failing To Authenticate From Process
+id: 9015385a-9c84-11eb-bef2-acde48001122
+version: 1
+date: '2021-04-13'
+author: Mauricio Velazco, Splunk
+type: batch
+datamodel:
+- Endpoint
+description: 'The following analytic identifies a source process name failing to authenticate
+  with multiple users. This behavior could represent an adversary performing a Password
+  Spraying attack against an Active Directory environment to obtain initial access
+  or elevate privileges. In this logic, the number of unique users is hardcoded to
+  ''10'' but can/should be customized to fit especific environments. Event 4625 documents
+  each and every failed attempt to logon to the local computer. This event generates
+  on domain controllers, member servers, and workstations. Logon Type 2 describes
+  an iteractive logon attempt.\
+
+  This detection will trigger on the potenfially malicious host, perhaps controlled
+  via a trojan or operated by an insider threat, from where a password spraying attack
+  is being executed. This could be a domain controller as well as a member server
+  orworkstation.\
+
+  The analytic''s returned fields allow analysts to investigate the event further
+  by providing fields like source process name, source account and attempted user
+  accounts.'
+search: ' `wineventlog_security` EventCode=4625 Logon_Type=2 Caller_Process_Name!="-"
```

Comment:
Maybe we can do specific error codes like for other events ... some interesting ones could be
Event ID 4625 status codes:

0xC0000064 User logon with misspelled or bad user account
0xC000006A User logon with misspelled or bad password
0XC000006D This is either due to a bad username or authentication information
0XC000006E Unknown user name or bad password.

Reply:
This is a good idea. Perhaps we can use this initial search to create two or three more specific ones with the different error codes.

However, I will leave that for a next iteration of this detection. We do want to push an initial version this week.